department-of-veterans-affairs/va.gov-cms
# VACMS-7619: Create field_va_health_connect_phone but keep it hidden

- Created "field_va_health_connect_phone" and configured it to be disabled on both manage form display and manage display.

## Description

closes #7619 

## Testing done

Passes all tests.

**Results of field help desk link test:**

>  —-—-—-—
Reply above this line.
Thanks for getting in touch with the Drupal CMS help desk.
Request description:
My VA health connect phone number is incorrect. ***THIS IS A TEST, PLEASE DISREGARD***
Please reply (to ivan.latorre@civicactions.com) to confirm this is working correctly. Thank you!
We'll take a look at your message, and will respond soon.
View request · Turn off this request's notifications
This is shared with Iván Latorre.
Powered by Jira Service Management 

## Screenshots
Fields:
![VACMS-7619_fields](https://user-images.githubusercontent.com/846871/150221541-36568064-bfcf-40ab-b872-4f9d5d1556fd.png)

Manage form display:
![VACMS-7619_manage_form_display](https://user-images.githubusercontent.com/846871/150221571-c425212b-4eda-47c6-86b8-01afbb76061b.png)

Manage display:
![VACMS-7619_manage_display](https://user-images.githubusercontent.com/846871/150221613-eb7def8d-f6d5-4744-bf44-53757e2cc3af.png)


## QA steps

As user administrator
1. [x] Go to `https://pr7677-2pdshfuwm3erspejwd4ymetrrtb7rv4s.ci.cms.va.gov//admin/structure/types/manage/health_care_region_page/fields` to verify and confirm the new `field_va_health_connect_phone` is listed there
2. [x] Go to `https://pr7677-2pdshfuwm3erspejwd4ymetrrtb7rv4s.ci.cms.va.gov//admin/structure/types/manage/health_care_region_page/form-display` to verify and confirm the new `field_va_health_connect_phone` is listed there and disabled in manage form display
3. [x] Go to `https://pr7677-2pdshfuwm3erspejwd4ymetrrtb7rv4s.ci.cms.va.gov//admin/structure/types/manage/health_care_region_page/display` to verify and confirm the new `field_va_health_connect_phone` is listed there and disabled in manage display
4. [x] Due to odd config changes... go to https://prod.cms.va.gov/central-texas-health-care and the tugboat version https://pr7677-2pdshfuwm3erspejwd4ymetrrtb7rv4s.ci.cms.va.gov/central-texas-health-care and do a side by side analysis to validate no changes.
5. [x] Due to odd config changes... go to https://prod.cms.va.gov/node/15023/edit and the tugboat version https://pr7677-2pdshfuwm3erspejwd4ymetrrtb7rv4s.ci.cms.va.gov/node/15023/edit and do a side by side analysis to validate no changes.


### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
